docs/UserGuide/Cluster/Cluster-Setup.md

```diff
@@ -19,376 +19,103 @@
 
 -->
 
-## Cluster setup
+## Cluster Setup
 
-You can follow this document to start an IoTDB cluster.
+This article is the setup process of IoTDB Cluster (0.14.0-preview1).
 
-## Prerequisite
-To use IoTDB, you need to have:
+## Environments
 
-1. Java >= 1.8 (Please make sure the environment path has been set)
+1. JDK>=1.8
 
-2. Set the max open files num as 65535 to avoid "too many open files" problem.
+2. Max open file 65535
 
-   
+3. Disable the swap memory
```

Comment:
Why should we disable the swap memory?

Reply:
To avoid system fluctuation even if the memory of IoTDB looks enough.